src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,340 @@
+import uuid
+
+from django.core.validators import MinLengthValidator, validate_integer
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        help_text=_("Unieke (technische) identificatiecode van de partij."),
+    )
+    nummer = models.CharField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[validate_integer],
+        max_length=10,
+    )
+    interne_notitie = models.CharField(
```

Comment:
```suggestion
    interne_notitie = models.TextField(
```

Laar max_length maar staan. Mag overal doorgevoerd worden waar max_length=1000.